.github/labeler.yml

```diff
@@ -0,0 +1,45 @@
+# Config file for pr-auto-label workflow
+
+# enable auto-labeler on issues, prs, or both.
+enable:
+  prs: true
+
+comments:
+  prs: |
+    Thanks for the contribution!
+    I have applied any labels matching special text in your title and description.
+
+    Please review the labels and make any necessary changes.
+
+# Labels is an object where:
+# - keys are labels
+# - values are objects of { include: [ pattern ], exclude: [ pattern ] }
+#    - pattern must be a valid regex, and is applied globally to
+#      title + description of issues and/or prs (see enabled config above)
+#    - 'include' patterns will associate a label if any of these patterns match
+#    - 'exclude' patterns will ignore this label if any of these patterns match
+labels:
+  'kind/CLEANUP':
+    include:
+      - '(/kind (chore|cleanup))|(chore|cleanup):'
```

Comment:
Agreed, we should skip these labels. 

I think I added them by looking at [this](https://github.com/prometheus/client_golang/blob/main/CHANGELOG.md#020--2015-02-23). But since our focus is user-facing changes, so we will remove these labels.